packages/core/src/lib/system-setting/system-setting.service.ts

```
@@ -0,0 +1,278 @@
+	/**
+	 * Retrieves settings from the database for a specific scope.
+	 *
+	 * @param {FindManyOptions} [request] - Optional query options for filtering settings.
+	 * @returns {Promise<Record<string, any>>} - A key-value pair object where keys are setting names and values are setting values.
+	 */
+	async getSettings(request?: FindManyOptions<SystemSetting>): Promise<Record<string, any>> {
+		let settings: SystemSetting[];
+
+		switch (this.ormType) {
+			case MultiORMEnum.MikroORM:
+				const { where, mikroOptions } = parseTypeORMFindToMikroOrm<SystemSetting>(request);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)